pettingzoo/utils/capture_stdout.py

```diff
@@ -14,10 +14,10 @@ class capture_stdout:
         data = var.getvalue()
     """
 
-    def __init__(self):
+    def __init__(self) -> None:
```

Comment:
I don’t think init needs this, afaik you never return anything from init

Reply:
Removed